src/Models/ApplicantModel.php

```diff
@@ -365,7 +368,21 @@ public function updateApplicantAdditionalFields(array $applicant)
                             `pleskHostingUrl` = :pleskHostingUrl,
                             `githubEducationLink` = :githubEducationLink,
                             `additionalNotes` = :additionalNotes,
-                            `chosenCourseId` = :chosenCourseId
+                            `chosenCourseId` = :chosenCourseId,
+                            `attitude` = :attitude,
+                            `averageScore` = :averageScore,
+                            `fee` = :fee,
+                            `signedTerms` = :signedTerms,
+                            `signedDiversitech` = :signedDiversitech,
+                            `signedNDA` = :signedNDA,
+                            `inductionEmailSent` = :inductionEmailSent,
+                            `checkedID` = :checkedID,
+                            `contactFormSigned` = :contactFormSigned,
+                            `dpName` = :dpName,
```

Comment:
given most other fields use fully labelled terms, perhaps dp isn't semantic enough? What does it mean? Display profile? It's not immediately obvious when you read it like inductionEmailSent is.